docs/src/documentation/05-development/04-migration-guides/01-v17.mdx

```diff
@@ -14,4 +14,10 @@ With this guide, we aim to walk through all the breaking changes and how they ca
 
 ### ColoredTwitter icon removed
 
-The `ColoredTwitter` icon was removed. The `Twitter` icon should be used instead.
+The `ColoredTwitter` icon was removed. The `X` icon should be used instead.
```

Comment:
Given this, I'm wondering if we should just bundle the removal with the `Twitter` icon too.

Reply:
I thought the same… We can… I have nothing against it. And we avoid the deprecation entirely?

Reply:
Agreed IRL that we will avoid the deprecation and just remove the icon in this release as well